server/settings/creator_plugins.py

```diff
@@ -182,7 +182,7 @@ class TrayPublisherCreatePluginsModel(BaseSettingsModel):
                     "type": "text",
                     "default": "",
                     "required_column": True,
-                    "validation_pattern": "^([a-zA-Z\\:\\ 0-9#._\\\\/]*)$"
+                    "validation_pattern": "^([a-zA-Z\\:\\ 0-9#-._\\\\/]*)$"
```

Comment:
This fixes so that `-` is allowed in input filenames?

Reply:
Actually, shouldn't `-` need to be escaped within `[]` for regex? (Using a backslash?) Otherwise it identifies a range, right? Like `[A-Za-z]`

Reply:
Or put the hyphen last or first in the brackets, like explained [here](https://stackoverflow.com/a/3697210)

Reply:
Yeah, you're right. It worked on my side but I think it was a pure coincidence that `9#-.` interpreted as the range between `#` and `.` covers`-`. Also took the opportunity to adjust `.` to be `\\.` as well as my understanding is that we want to allow points, not any single character.